R/ecoretriever.R

```diff
@@ -136,11 +136,37 @@ datasets = function(){
   system('retriever ls', intern = TRUE) 
 }
 
+#' Update the retrievers download script to the most recent official release.
+#' 
+#' This function will check if the version of retriever scripts in your local
+#' directory ~/.retriever/scripts/ is up-to-date with the most recent offical
+#' retriever release. Note it is possible that even more updated scripts exist
+#' at the retriever repository https://github.com/weecology/retriever/tree/master/scripts 
+#' that have not yet been incorperated into an official release, and you should 
+#' consider checking that page if you have any concerns. 
```

Comment:
This is slightly complicated as I was just explaining to Dave Harris in another issue: https://github.com/weecology/retriever/issues/297#issuecomment-99978196. So, this is sort of like documenting a bug, which seems a little weird, but I'm also OK with leaving it in if you'd prefer.